.github/workflows/cromwell_unit_tests.yml

```diff
@@ -0,0 +1,86 @@
+name: 'Cromwell unit tests'
+
+#This github action runs all of Cromwell's unit tests.
+#A matrix strategy is used so that each subproject can be unit tested in parallel with all other subprojects. 
+
+#This is what shows up in the github workflows page as the title. 
+run-name: ${{ github.actor }} running Cromwell sbt unit tests.
+
+#What will trigger the workflow to run. 
+on:
+  workflow_dispatch: #Manual trigger from GitHub UI
+  push:
+  pull_request: 
```

Comment:
I'm not sure we want both `push` and `pull_request` here - I think it's running both when a PR is opened and also on each push. This results in two entries in the GHA list associated with the PR for each test. This is something we can iterate on over time, as we learn how expensive these tests are (is it better to run them way more often than usual or need to manually trigger them sometimes?). To start, maybe `push` and `workflow_dispatch` only?

Reply:
Done. Makes sense to me.